onnxruntime/test/providers/cpu/activation/activation_op_test.h

```diff
@@ -105,7 +105,7 @@ class ActivationOpTest : public ::testing::Test {
     std::random_device rd;
     std::mt19937 gen(rd());
     std::uniform_real_distribution<float> dist(low, high);
-    std::vector<std::size_t> batch_size_list = {1, 2, 4, 9, 100000};
+    std::vector<std::size_t> batch_size_list = {1, 2, 4, 9, 10000};
```

Comment:
https://github.com/microsoft/onnxruntime/blob/34cd2e8ed8688c42d00adda1d260ac787f76bf29/onnxruntime/core/providers/coreml/builders/helper.cc#L81

Reply:
If the value is too large does the test fail? I would have expected it falls through to the CPU EP.

Instead of reducing the 100,000 (there may be some existing code that needs a value that large to exercise it) could we instead add another smaller one? Maybe 1,000 would be fine.

Reply:
Yeah, this falls back to CPU EP, But we should have some tests go through CoreML EP.

I set a different values and coreml and others. comments added as well.